src/main.cpp

```diff
@@ -20,11 +19,15 @@ int main()
     std::cout<<num2;
     std::cout<<std::endl;
 
-    std::cout<<(num2/num3)<<std::endl;
+    try {
+        std::cout<<(num2/num3)<<std::endl;
+    } catch(const std::logic_error& e) {
```

Comment:
Since now we are throwing `std::logic_error` we can catch by exact type.
Otherwise we could keep it as it was `throw ("Divide by Zero");` and use `catch(...)`